File: OMKARAUTOAPP/src/app/models/dailyloadingrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Dailyloadingrptmodel } from "./dailyloadingrptmodel";

export class Dailyloadingrptlistmodel {
    dailyLoadingRptsList: Dailyloadingrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: OMKARAUTOAPI/FleetTrans/Models/DailyLoadingRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Shared.Models;

namespace FleetTrans.Models
{
    public class DailyLoadingRptListModel
    {
        public List<DailyLoadingRptModel> DailyLoadingRptsList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
